cypress/support/elements/register_add_student_page_elements.js

```diff
@@ -7,7 +7,18 @@ const registerAddNewStudentPageElements = {
   SUBMIT_BUTTON : 'div[class^=modal] form tr:nth-child(5) input',
   DIALOG_TEXT : 'div[class^=modal] div[class^=dialog] div[class^=title]',
   DIALOG_ADD_ANOTHER_STUDENT_BUTTON : 'div[class^=modal] div[class^=dialog] button:nth-child(1)',
-  DIALOG_CANCEL_BUTTON : 'div[class^=modal] div[class^=dialog] button:nth-child(2)'
+  DIALOG_CANCEL_BUTTON : 'div[class^=modal] div[class^=dialog] button:nth-child(2)',
+
+  BTN_REGISTER_USER: 'a[href=\"/signup\"]',
+  BTN_I_AM_STUDENT: 'button[value=\"student\"]',
+  TXT_FIRST_NAME: 'div.signup-form dd.first-name-wrapper input[type=\"text\"]',
+  TXT_LAST_NAME: 'div.signup-form dd.last-name-wrapper input[type=\"text\"]',
+  TXT_PASSWORD: 'div.signup-form dd div.password input[type=\"password\"]',
+  TXT_CONFIRM_PASSWORD: 'div.signup-form dd div.password_confirmation input[type=\"password\"]',
+  BTN_SUBMIT_BUTTON: 'div.signup-form div.submit-button-container button.submit-btn',
+  TXT_CLASS_WORD: 'div.signup-form div.class_word input[type=\"text\"]',
+  LBL_SIGNUP_SUCCESS: 'div.signup-form div.registration-complete.student div p',
```

Comment:
These page elements seem to be from the signup page for the student. Can we please move them to the signup_page_elements.js? This file represents page elements within the teacher account within a class when the teacher registers and adds students directly to the class. I agree the name for this file is confusing, we should probably change it to something like teacher_register_add_student_page-elements.js